src/org/traccar/database/PermissionsManager.java

```diff
@@ -197,6 +198,11 @@ public boolean isDeviceReadonly(long userId) {
         return user != null && user.getDeviceReadonly();
     }
 
+    public boolean isLimitCommands(long userId) {
```

Comment:
This sounds a bit weird. Should we rename all of the methods from `is` to `get`?

Reply:
Then may be `getUser...` to not confuse with combined user/server

Reply:
Sounds good to me.